### PR TITLE
Arabienko rpa s4 java h2 t1

### PR DESCRIPTION
**Pull request by arabienko**

### CODE REVIEW

File: homework/java/arabienka/Lesson2/Task1.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Количество кораблей в порту -countShipInPort
+Счетчик целых чисел - countIntNumb
+Первое число - oneNumb
```
Comment (vitalikulsha) [inline homework/java/arabienka/Lesson2/Task1.md:3]: firstNumber

Comment (arabienko) [reply homework/java/arabienka/Lesson2/Task1.md:3]: ok

File: homework/java/arabienka/Lesson2/Task1.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Ответ на вопрос - answQuestion
+Случайное число - randomNumb
+Генератор случайных чисел genRundomNumb
```
Comment (vitalikulsha) [inline homework/java/arabienka/Lesson2/Task1.md:6]: genRundonNumb -> genRandomNum

File: homework/java/arabienka/Lesson2/Task1.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Цвет автомобиля - colorCar
+Скорость движения автомобиля - speedGoCar
+Введеное число пользователем - inNumbUser
+Наибольшее число - bigNumb
+Наименьшее число - smallNumb
+Левая граница диапазона чисел - leftBoardRangNumb
+Правая граница диапазона чисел - rightBoardRangNumb
+Загаданное пользователем число - misticUserNumber
+Вариант ответа - varAnswer
+Банковский счет - banBill
```
Comment (vitalikulsha) [inline homework/java/arabienka/Lesson2/Task1.md:16]: bankBill

**Review by vitalikulsha — APPROVED** [review]

**Review by FancyTanya — APPROVED** [review]

File: homework/java/arabienka/Lesson2/Task1.md (hunk 3 above)
Comment (arabienko) [reply homework/java/arabienka/Lesson2/Task1.md]: ok

**Review by Fenix2021 — APPROVED** [review]

**Review by vitalikulsha — APPROVED** [review]

**Pull request merged**